de.ifw_dresden.nagstamon.yml

```diff
@@ -0,0 +1,145 @@
+        sha256: 44e5a4e1ad1070c7cd46959a0d6c2a99588b66a6dad6fd685463b486c04855ea
+      - type: file
+        url: https://files.pythonhosted.org/packages/98/61/fcd53201a23dd94a1264c29095821fdd55c58b4cd388dc7115e5288866db/PyQt5-5.12.1-5.12.2-cp35.cp36.cp37.cp38-abi3-manylinux1_x86_64.whl       
+        sha256: d4e88208dfd017636e4b1f407990d0c3b6cf47afed6be4f2fb6ca887ef513e4b
+      - type: file
+        url: https://files.pythonhosted.org/packages/2e/ee/c6efd28dcb65e990411be50a46c9757f12202df94cfee73689223bbc2cf0/PyQt5_sip-4.19.15-cp37-cp37m-manylinux1_x86_64.whl
+        sha256: 31a59f76168df007b480b9382256c20f8898c642e1394df2990559f0f6389f66
+      - type: file
+        url: https://files.pythonhosted.org/packages/b7/cf/1ea0f5b3ce55cacde1e84cdde6cee1ebaff51bd9a3e6c7ba4082199af6f6/wheel-0.33.1.tar.gz
+        sha256: 66a8fd76f28977bb664b098372daef2b27f60dc4d1688cfab7b37a09448f0e9d
+      - type: file
+        url: https://files.pythonhosted.org/packages/54/85/514ba3ca2a022bddd68819f187ae826986051d130ec5b972076e4f58a9f3/setuptools_scm-3.2.0.tar.gz
+        sha256: 52ab47715fa0fc7d8e6cd15168d1a69ba995feb1505131c3e814eb7087b57358
+      - type: file
+        url: https://files.pythonhosted.org/packages/35/35/da1123673c54b6d701453fcd20f751d6a1fae43339b3993ae458875576e4/pytoml-0.1.20.tar.gz
+        sha256: ca2d0cb127c938b8b76a9a0d0f855cf930c1d50cc3a0af6d3595b566519a1013
+      - type: file
+        url: https://files.pythonhosted.org/packages/84/f4/5771e41fdf52aabebbadecc9381d11dea0fa34e4759b4071244fa094804c/docutils-0.14.tar.gz
+        sha256: 51e64ef2ebfb29cae1faa133b3710143496eca21c530f3f71424d77687764274
+      - type: file
+        url: https://files.pythonhosted.org/packages/1f/87/9ea76ab4cdf1fd36710d9688ec36a0053067c47e753b32272f952ff206c5/flit-1.3.tar.gz
+        sha256: 6f6f0fb83c51ffa3a150fa41b5ac118df9ea4a87c2c06dff4ebf9adbe7b52b36   
+      - type: file
+        url: https://files.pythonhosted.org/packages/f9/a5/5dacebf93232a847970921af2b020f9f2a8e0064e3a97727cd38efc77ba0/intreehooks-1.0.tar.gz
+        sha256: 87e600d3b16b97ed219c078681260639e77ef5a17c0e0dbdd5a302f99b4e34e1
+
+  - name: setup
+    buildsystem: simple
+    build-commands:
+      - install -D nagstamon.sh /app/bin/nagstamon
+      - install -Dm644 nagstamon-icon-64x64.png /app/share/icons/hicolor/64x64/apps/de.ifw_dresden.nagstamon.png
+      - install -Dm644 nagstamon-icon-128x128.png /app/share/icons/hicolor/128x128/apps/de.ifw_dresden.nagstamon.png
+      - install -Dm644 de.ifw_dresden.nagstamon.desktop -t /app/share/applications
+      - install -Dm644 de.ifw_dresden.nagstamon.appdata.xml -t /app/share/metainfo
+      - python3 setup.py install
+    build-options:
+      build-args:
+        - --share=network
+    sources:
+      - type: git
+        url: https://github.com/HenriWahl/Nagstamon.git
+        tag: '3.2.1'
+      - type: file
```

Comment:
Ideally appdata, desktop and icons should be contributed or taken from upstream repo.